server.js

```diff
@@ -563,38 +569,41 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
         try {
           var data = JSON.parse(buffer);
 
-          var percentage = data[0].msr[0].val;
+          var value = data[0].msr[0].val;
 
-          if (percentage === undefined) {
+          if (value === undefined) {
             badgeData.text[1] = 'unknown';
             sendBadge(format, badgeData);
             return;
           }
 
-          if (metric === 'tech_debt') {
+          if (metric === 'lines') {
+            badgeData.text[1] = value;
```

Comment:
Makes sense, I didn't notice the function and just used the name that was already there.  I will change it.